.github/workflows/enable-automerge.yml

```diff
@@ -2,12 +2,6 @@ name: Automerge
 
 on:
   pull_request_target:
-    types:
-      - opened
-      - synchronized
-      - reopened
-      - edited
-      - ready_for_review
```

Comment:
I *think* the proper format for this would be: `on: [ pull_request_target ]`?  At least, that is how I have seen it in the past (and YML is ridiculous).